src/BulletDynamics/Vehicle/btRaycastVehicle.cpp

```diff
@@ -551,46 +561,43 @@ void	btRaycastVehicle::updateFriction(btScalar	timeStep)
 			m_forwardImpulse[i] = btScalar(0.);
 
 		}
-	
+
```

Comment:
what is happening here? Is this a code change or just formatting? If formatting, please undo.

Reply:
It's hard to see, but there were some unnecessary curly brackets so I removed them and unindented the rest. It's clearer to see if you add '?w=1' to the url for the PR in order to hide whitespace changes. I didn't really pay attention to this part of the diff as I was preparing it, but if you'd like I can either revert it or go back through and make sure it's just a curly bracket removal and indentation change, and that there are no lines added or removed and no tabs changed to spaces.